bundle/run/job_options.go

```diff
@@ -72,6 +77,21 @@ func (o *JobOptions) Validate(job *resources.Job) error {
 		return errors.New("the job to run does not define job parameters; specifying job parameters is not allowed")
 	}
 
+	if len(o.only) > 0 {
+		for _, task := range o.only {
+			found := false
+			for _, t := range job.Tasks {
+				if t.TaskKey == task {
+					found = true
+					break
+				}
+			}
+			if !found {
+				return fmt.Errorf("task %#v not found in job %#v", task, job.Name)
```

Comment:
Please check with the team if they are planning to overload these keys with something like "task1>" to mean task1 and all tasks that depend on it. I recall that being under consideration. If so, then we should either not validate these keys at all (and perhaps only autocomplete), or only do validation we know is forward-compatible.

Reply:
Confirmed that it will eventually be extended to `task1.table1` like syntax, making a validation forward compatible

Reply:
https://github.com/databricks/cli/pull/3388/commits/f7fad0598278ccc666b80e84af2c574e3fc81890